worker/src/dns.rs

```diff
@@ -0,0 +1,83 @@
+use crate::SPRUCE_DIDWEB;
+use anyhow::{anyhow, Result};
+use chrono::{SecondsFormat, Utc};
+use serde::Deserialize;
+use serde_json::json;
+use ssi::{blakesig::hash_public_key, jwk::JWK, vc::Credential};
+use url::Url;
+use uuid::Uuid;
+
+#[derive(Deserialize, Debug)]
+pub struct DnsResponse {
+    pub Answer: Vec<AnswerResponse>,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct AnswerResponse {
+    pub name: String,
+    pub data: String,
+}
+
+pub fn extract_dns_signature(tweet: String) -> Result<(String, String)> {
+    let mut sig_target = "".to_string();
+    for line in tweet.split('=').collect::<Vec<&str>>() {
+        if line.starts_with("sig:") {
```

Comment:
`sig:` is only there in other verifications to identify the signature, but you already have the field's name to do that.

Reply:
Removed